samples/hcls_2015.shex

```diff
@@ -97,7 +97,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
     dct:title rdf:langString,
     ###dct:alternative rdf:langString, is a MAY property
     dct:description rdf:langString,
-    (dct:creator IRI | pav:createdBy IRI)+,
+    (dct:creator IRI, pav:createdBy IRI)+,
```

Comment:
Alasdair talked about this in the last meeting. The "or" in the document refers to inclusive or, rather than exclusive or. So dct:creator can exist as well as pav:createdBy, and there can be multiple of them (or none). Based on my understanding from the wiki, that is correct. Same goes for the rest of these ones.